src/claimtrie.cpp

```diff
@@ -1112,20 +1263,23 @@ bool CClaimTrie::WriteToDisk()
 
 bool CClaimTrie::InsertFromDisk(const std::string& name, CClaimTrieNode* node)
 {
-    if (name.size() == 0)
-    {
+    if (name.size() == 0) {
         root = *node;
         return true;
     }
+    const std::string& newName = (fNormalize ? normalizeClaimName(name) : name);
+    if (fNormalize && newName != name) {
+        delete node;
```

Comment:
This line looks suspicious to me; what if the previous node had claims on it? It appears that we don't keep those. Is this where we're losing claims? We need to change them so that all the original ones still complete; don't throw them away.